db/migrate/20171116151437_add_column_to_user.rb

```diff
@@ -0,0 +1,13 @@
+class AddColumnToUser < ActiveRecord::Migration[5.1]
+  def change
+  	add_column :users, :name, :string
+  end
+
+  def up
+    drop_table :instructors
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+end
```

Comment:
Since we're still so early in development, I would suggest editing the `DeviseCreateUsers` migration and dropping + creating + migrating the whole database again. We can't do that when we're in production and have live data coming in, but right now this is fine. The upside of doing this is that our migration's can serve as a record for any developer coming into this app, who will be able to easily see all the columns generated for this table with ease, instead of having to open each migration up to see the paper-trail of migrations that eventually form the table.